scripts/form-validation.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+
+function validateForms(path) {
+  var formFolders = fs.readdirSync(path);
+  var passedValidation = true;
+  formFolders.forEach(function(folderName) {
+    var folderPAth = path + folderName + '/';
+    var files = fs.readdirSync(folderPAth);
+    var filterFnames = files.filter(
+      fName => fName.includes('.xml') || fName.includes('.xlsx')
+    );
+    for (var i = 0; i < filterFnames.length; i += 2) {
+      var xml = folderPAth + filterFnames[i + 1];
+      var xlsx = folderPAth + filterFnames[i];
+      if (
+        validateNamesDoNotMatch(xlsx, xml) ||
+        validateModifiedTimesDoNotMatch(xml, xlsx)
+      ) {
+        passedValidation = false;
+        return;
+      }
+    }
+  });
+  return passedValidation;
+}
+
+function validateNamesDoNotMatch(xlsxName, xmlName) {
```

Comment:
Use positive function names, eg: `validateNamesMatch`. Otherwise you end up with double negatives which can make the code really hard to read, eg: `!notFalse(true) || !notTrue(false)`